authorizer/task_test.go

```diff
@@ -577,17 +581,15 @@ from(bucket:"holder") |> range(start:-5m) |> to(bucket:"holder", org:"thing")`
 	}
 }
 
-func setup(t *testing.T) (*tenant.Service, influxdb.OnboardingService) {
+func setup(t *testing.T, useHashedTokens bool) (*tenant.Service, influxdb.OnboardingService) {
 	t.Helper()
 
 	store := newStore(t)
 
 	svc := tenant.NewService(tenant.NewStore(store))
 
-	authStore, err := authorization.NewStore(store)
-	if err != nil {
-		t.Fatal(err)
-	}
+	authStore, err := authorization.NewStore(context.Background(), store, useHashedTokens)
+	require.NoError(t, err)
```

Comment:
Context string for NoError

Reply:
Doesn't add any value and its for the line above it.